src/common/entryPointHandler.ts

```diff
@@ -1,9 +1,14 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for details.
 
+
+import {ErrorHelper} from "../common/error/errorHelper";
+import {InternalError} from "../common/error/internalError";
+import {InternalErrorCode} from "../common/error/internalErrorCode";
 import {TelemetryHelper} from "../common/telemetryHelper";
 import {Telemetry} from "../common/telemetry";
-import {Log} from "../common/log";
+import {Log} from "../common/log/log";
+import {OutputChannelLogger} from "../common/log/loggers";
 import {OutputChannel} from "vscode";
```

Comment:
Same for this common class, can we pass the logger around instead of the output channel?